test/drenv/envfile.py

```diff
@@ -3,9 +3,32 @@
 
 import os
 import copy
+import platform
+import logging
```

Comment:
Unused import (reported by flake8)

You can run this locally:
```
make flake8
```

Reply:
Done, thanks!